.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
could you remove this from the PR. sorry, just wanted to move it to help get the extensions loaded initially, but there is another PR that is set to move this file into the .vscode folder